aw_client/config.py

```diff
@@ -0,0 +1,16 @@
+from configparser import ConfigParser
+
+from aw_core import dirs
+from aw_core.config import load_config
+
+default_client_config = ConfigParser()
+default_client_config["server"] = {
+    "hostname": "localhost",
+    "port": 5600,
+}
+default_client_config["server-testing"] = {
+    "hostname": "localhost",
+    "port": 5666
+}
+
+client_config = load_config("aw-client", default_client_config)
```

Comment:
I still think that importing a global variable is an anti-pattern. But I'll let this one slide.